core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -721,6 +721,11 @@ public NativeImageInvokerInfo build() {
                             .map(p -> p.getSecurityProvider())
                             .collect(Collectors.joining(","));
                     nativeImageArgs.add("-H:AdditionalSecurityProviders=" + additionalSecurityProviders);
+                    // Work around https://github.com/quarkusio/quarkus/issues/21374
```

Comment:
Yeah, I'm wondering if we should have a `AddJPMSExportBuildItem` given there's a good change we will have more of it.

Reply:
I added the build item and updated the PR. Please have a look. The only thing missing now is the isJava17 check, but that is OK since the export doesn't cause any trouble in Java 11.